src/Pages/About.ts

```diff
@@ -1,18 +1,18 @@
 import { connect } from "react-redux";
-import { HomeProps, Home } from "./Home/index";
+import { AboutProps, About } from "./About/index";
 import { ThunkDispatch } from "../Helpers/StateHelpers";
-import { updateCounter } from "../Modules/Home";
+import { updateCounter } from "../Modules/About";
```

Comment:
Wiem, że to ja zostawiłem ten smród, ale można usunąć ten `updateCounter`.

Reply:
A co to w ogóle było?

Reply:
A tak sobie testowałem czy redux działa.